src/GitHubExtension/DeveloperId/DeveloperIdProvider.cs

```diff
@@ -102,6 +103,39 @@ public IAsyncOperation<IDeveloperId> LoginNewDeveloperIdAsync()
         }).AsAsyncOperation();
     }
 
+    private DeveloperId LoginNewDeveloperIdWithPAT(Uri hostAddress, SecureString personalAccessToken)

```

Comment:
Just had a question about how this works. I know this isn't used now but if this method throws, where in Dev Home would it bubble up to?

Reply:
This will only be called from LoginUI adaptive card interaction. These exceptions don't bubble up to DevHome currently.